### PR TITLE
Json parametrize more2

### PR DESCRIPTION
**Pull request by WillAyd**

### CODE REVIEW

File: pandas/tests/io/json/test_pandas.py [hunk 1]
```diff
@@ -730,34 +718,33 @@ def test_reconstruction_index(self):
...
         tm.assert_frame_equal(result, df)
 
-    def test_path(self, float_frame):
+    def test_path(self, float_frame, int_frame, datetime_frame):
         with tm.ensure_clean("test.json") as path:
             for df in [
                 float_frame,
-                self.intframe,
-                self.tsframe,
-                self.mixed_frame,
```
Comment (WillAyd) [inline pandas/tests/io/json/test_pandas.py]: I just removed `mixed_frame` here as this is its only use in the module; doesn't seem important to re-create, and in its current state it was really the same thing as `float_frame` anyway

**Comment by jbrockmendel** [comment]
LGTM

**Review by jschendel — APPROVED** [review]
lgtm, one small comment

File: pandas/conftest.py [hunk 2]
```diff
@@ -526,6 +526,66 @@ def empty_frame():
...
+    y3bpw4nwIp  0 -1  0  0
+    H0RdLLwrCT  1  1  0  0
+    rY82K0vMwm  0  0  0  0
+    1OPIUjnkjk  2  0  0  0
+
+    [30 rows x 4 columns]
+    """
+    df = DataFrame({k: v.astype(int) for k, v in tm.getSeriesData().items()})
+    # force these all to int64 to avoid platform testing issues
+    return DataFrame({c: s for c, s in df.items()}, dtype=np.int64)
```
[the remark this answers is not in the document]
Comment (WillAyd) [reply pandas/conftest.py]: yea I agree - nice catch!

[the remark this answers is not in the document]
Comment (jbrockmendel) [reply pandas/conftest.py]: i assumed this had to do with the uniqueness mentioned in the docstring

File: pandas/conftest.py [hunk 3]
```diff
@@ -526,6 +526,64 @@ def empty_frame():
...
+    return DataFrame(tm.getSeriesData()).astype("int64")
+
+
+@pytest.fixture
+def datetime_frame():
+    """
+    Fixture for DataFrame of floats with DatetimeIndex
+
+    Columns are ['A', 'B', 'C', 'D']
+
```
Comment (jreback) [inline pandas/conftest.py]: doesn't this duplicate some existing fixtures?

File: pandas/conftest.py [hunk 4]
```diff
@@ -526,6 +526,64 @@ def empty_frame():
     return DataFrame()
 
 
+@pytest.fixture
+def int_frame():
+    """
```
Comment (jreback) [inline pandas/conftest.py]: does this duplicate?

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by WillAyd** [comment]
@jreback no to both comments - these are the only fixtures with those names

**Comment by jreback** [comment]
> @jreback no to both comments - these are the only fixtures with those names

kk

**Review by jreback — APPROVED** [review]

**Pull request merged**